bears/general/InvalidLinkBear.py

```diff
@@ -75,9 +79,17 @@ def run(self, filename, file,
         :param timeout:          Request timeout period.
         :param link_ignore_regex:     A regex for urls to ignore.
         :param follow_redirects: Set to true to autocorrect redirects.
+        :param network_timeout:  Number of seconds to wait for the reponse from
+                                 server.
+        :param network_url:      The root url of the server whose timeout has
+                                 to be changed.
         """
+        if bool(network_timeout) and bool(network_url):
+            urlparse(network_url).netloc
+
         for line_number, link, code in InvalidLinkBear.find_links_in_file(
-                file, timeout, link_ignore_regex):
+                file, timeout=timeout if not bool(network_timeout) else
```

Comment:
I thought it doesn't, so I updated the PR, initially it was just the variable. But then i thought it'd give TypeError